0015-authority.md

```diff
@@ -0,0 +1,478 @@
+- Feature Name: Authority
+- Start Date: 2021-05-19
+- RFC PR: [Unity-Technologies/com.unity.multiplayer.rfcs#0015](https://github.com/Unity-Technologies/com.unity.multiplayer.rfcs/pull/0015)
+- Issue: [Unity-Technologies/com.unity.multiplayer#0000](https://github.com/Unity-Technologies/com.unity.multiplayer/issues/0000)
+
+# Summary
+[summary]: #summary
+
+The authority concept is an abstraction layer for gameplay logic on top of MLAPI. It allows high level components to write game play code without the need of considering the underlying network model. Authority does not directly affect other parts of the framework.
+
+# Motivation
```

Comment:
I can see this proposal from a technical perspective but I'm struggling to understand what are the real motivations behind this. Instead of "here's a concept that might be useful", I'd like to see something like "here's a concept that _is_ useful because we will use it to solve X problem in Y context".
I got "what?" and "how?" but I couldn't get "why?" and "when?".

Reply:
I cannot implement NetworkPhysics  without this. We cannot  provide NetworkTransform or NetworkAnimator without this. This is solving real problems we have right now.

Reply:
Do you mind mentioning and explaining those real problems within the RFC?

Reply:
I will add more information to the motivation section to explain this.